### PR TITLE
Fix bounds setting

### PR DESCRIPTION
**Pull request by trevorgerhardt**

### CODE REVIEW

File: lib/components/analysis/regional.js [hunk 1]
```diff
@@ -40,11 +42,29 @@ export default class RegionalAnalysis extends React.PureComponent {
...
   }
 
+  _isComplete () {
+    const p = this.props
+    return !!p.analysis && (p.analysis.status == null || p.analysis.status.complete === p.analysis.status.total)
+  }
+
+  _getAnalysisBounds () {
+    const {north, west, width, height, zoom} = this.props.analysis
+    const nw = LeafletMap.prototype.unproject([west + 1, north], zoom)
```
Comment (ansoncfit) [inline lib/components/analysis/regional.js]: +1 ?

File: lib/components/analysis/advanced-settings.js [hunk 2]
```diff
@@ -93,18 +92,16 @@ export default class AdvancedSettings extends Component {
...
+    const {profileRequest, regionBounds, regionalAnalyses} = this.props
+    const {bounds} = profileRequest
 
     // figure out which is selected
     let selected
-    if (
-      !regionalAnalysisBounds ||
-      boundsEqual(regionalAnalysisBounds, regionBounds)
-    ) {
+    if (!bounds || boundsEqual(bounds, regionBounds)) {
```
Comment (ansoncfit) [inline lib/components/analysis/advanced-settings.js]: Where is boundsEqual defined?

File: lib/components/analysis/regional.js (hunk 1 above)
Comment (abyrd) [reply lib/components/analysis/regional.js]: Yeah it's strange to see west + 1 but not north + 1...

Comment (trevorgerhardt) [reply lib/components/analysis/regional.js]: Copied code. No comments on the original either. 🤔

**Pull request merged**